frontend/portals/StickyMedia/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import { scrolledTopOffset } from '../../config';
+
+const styles = {
+  wrapper: css({
+    position: 'sticky',
+    top: scrolledTopOffset,
+    zIndex: 99,
+  }),
+};
+
+/**
+ * @param {Object} props Props
+ * @return {JSX}
+ */
+const StickyMedia = ({ children }) => (
+  <div className={styles.wrapper}>
```

Comment:
You can use React.cloneElement and mutate a `className` prop of children instead of adding extra dom node

Reply:
kept it since i added new portal